channels/fr.m3u

```diff
@@ -21,8 +23,16 @@ https://bfmlyon-lh.akamaihd.net/i/BFMLYON_KBS@797041/master.m3u8
 https://bfmparishdslive-lh.akamaihd.net/i/BFMPARIS_KB@429747/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="" group-title="",BFMTV (720p)
 https://bfmtvalive1-a.akamaihd.net/1992d859ea9c46a4a5108ec10e1385b5/eu-central-1/876450610001/7b4151e1e2434a7cacdb9936db7a7910/playlist_ssaiM.m3u8
+#EXTINF:-1 tvg-id="" tvg-name=" tvg-language="French" tvg-logo="https://i.imgur.com/mihujKt.jpg" group-title="News",BFMTV
+https://bcovlive-a.akamaihd.net/fb3647b53c1a4c278eaecf83230ee4c5/eu-central-1/876450610001/profile_0/chunklist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://i.imgur.com/YFEqBS7.png" group-title="",BipTV
 https://biptv.tv/live/biptvstream_orig/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/es/Boing.es.png" group-title="Kids",BOING
+http://cdn.webtv4.cdnfr.orange.fr/hs/HALO5/hls/live/Boing_live/01.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/br/Boomerang.br.png" group-title="Kids",Boomerang
+http://cdn.webtv4.cdnfr.orange.fr/hs/HALO5/hls/live/Boomerang_live/01.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="French" tvg-logo="https://static.epg.best/fr/Cherie25.fr.png" group-title="",Chérie 25
+https://cherie25hls-lh.akamaihd.net/i/cherie25hls_1@579096/master.m3u8
```

Comment:
The link doesn't work.
<img width="496" alt="Captura de pantalla 2021-01-25 a las 17 22 46" src="https://user-images.githubusercontent.com/7253922/105718171-031a7080-5f32-11eb-81d9-4f07f4e08af5.png">

Reply:
For both ones ?

Reply:
No, only: https://cherie25hls-lh.akamaihd.net/i/cherie25hls_1@579096/master.m3u8

Reply:
Maybe it's geo-locked. I'll check when I can on streamtest.

Reply:
Found out it was geolocked. Deleted.